src/components/data-list/DateValueGetter/index.ts

```diff
@@ -0,0 +1,9 @@
+const DateValueGetter = (params: any) => {
+  const createdAt = params.value as string | undefined;
```

Comment:
- It is confusing that you use createdAt const name inside a generic function which works with any date fields (it could be birthday, updatedAt, etc.)